rpc/documentation/api.md

```diff
@@ -1476,3 +1478,27 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
```

Comment:
Sorry, I was thinking of getting to this once all the RPC commands are in. Will update that.